src/web/app/components/question-types/question-statistics/contribution-question-statistics/contribution-question-statistics.module.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { ContributionRatingsListComponent } from './contribution-ratings-list.component';
+import { ContributionComponent } from './contribution.component';
+
+/**
+ * Module for displaying data in a sortable table
+ */
+@NgModule({
+  declarations: [ContributionComponent, ContributionRatingsListComponent],
+  imports: [
+    NgbModule, CommonModule,
+  ],
+  exports: [
+    ContributionComponent, ContributionRatingsListComponent,
+  ],
+})
+export class ContributionQuestionStatisticsModule { }
```

Comment:
Is defining the module really necessary?

Reply:
Not really necessary, but I think it helps make the code more organized. Especially since I added a new `ContributionRatingsListComponent`. Now's there's 3 components used by the contribution statistics. Grouping them together would be nice.